### ISSUE TITLE
A11y_Playwright.dev_Contrast Contrast: Color contrast ratio fails for the Controls present in the "Chosen by companies and open source projects" section.

### ISSUE THREAD

**Issue by Raghu-addepalli** [microsoft/playwright.dev#1271]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS;#A11yTCS;#Playwright releases;#P1_Playwright.dev_Web_Jan2024;#A11ySev2;#A11YWCAG2.1;#ChromiumEdge;#DesktopWeb;#FTP;#WCAG1.4.11;#Non Text Contrast;#Win11;#P1_WCP

### Environmental Details:
App Name: Playwright.dev
#URL: https://playwright.dev/java/

### OS & Browser Details:
Microsoft Windows 11 Enterprise
Version 22621.3007
Microsoft Edge
Version 120.0.2210.144 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL: https://playwright.dev/java/.
2. Open Accessibility Insights for Windows Tool.
3. Now check the Color Contrast for the Controls present in the "Chosen by companies and open source projects" section is greater than or equal to 3:1 or not.

### Actual results:
Color contrast ratio "2.098" fails for the Controls present in the "Chosen by companies and open source projects" section.

### Expected results:
Color contrast ratio fails for the Controls present in the "Chosen by companies and open source projects" section should be greater than or equal to 3:1 required.

### User Impact:
Low Vision users will not get about the controls if the Non-Text color contrast ratio is less than required.

### Attachment:
![CC](https://github.com/microsoft/playwright.dev/assets/91586743/1ac925c0-784a-4a4f-995c-70c1a6f183ce)

**Comment by mxschmitt** [comment]
This should be fixed.

**Comment by Raghu-addepalli** [comment]
verified the issue in below environment issue is Fixed and closing the bug.

https://playwright.dev/java/

![1271_Fixed](https://github.com/microsoft/playwright.dev/assets/91586743/34303017-2328-4e85-b46f-9033f0be5091)
